### ISSUE TITLE
Any plan to move to WebExtensions ?

### ISSUE THREAD

**Issue by temsa** [bakulf/priv8#23]
Hi :)

Still fond of priv8, yet I'm using e10s for months so I cannot use it. Would it be possible to rewrite this addon using WebExtensions so it could be available both in e10s AND FirefoxOS ? :)

**Comment by bakulf** [comment]
Hi Temsa, I'm not planning to rewrite this addon with WebExtensions. Instead, I'm collaborating with the security team of Mozilla in order to have something similar to priv8, but native in firefox.
The project is called containers: https://wiki.mozilla.org/Security/Contextual_Identity_Project/Containers
We are getting closer and closer. And trust me when I say that this is amazing!!

**Comment by temsa** [comment]
Great ! I had discussed with some mozillians last June in order to help design a GUI / Story board for this, yet I did not finish the job...

I was planning to integrate it with personas so you would immediately know in which sandbox you were (and easily change it). I would also have integrated the choice directly on a link for containers bound to a website(i.e. Facebook Container) maybe as a contextual menu with a default activated like 3s later.

Hope those can be done too (at least as an add-on) rather than just the tab highlight which is still great :)
